webpack.config.js

```diff
@@ -21,4 +21,5 @@ module.exports = {
         },
       ],
     },
-  }
\ No newline at end of file
+  }
+    
```

Comment:
Line 7 has some whitespaces, delete those whitespaces or tabs, without deleting line 7. Or maybe you have two blank lines, and it is just needed one.